census_view.cpp

```diff
@@ -1431,7 +2374,23 @@ void CensusView::UponRightClick(wxDataViewEvent& e)
     delete census_menu;
 }
 
-void CensusView::UponValueChanged(wxDataViewEvent&)
+void CensusGridView::UponRightClick(wxGridEvent&)
+{
+    std::unique_ptr<wxMenu> census_menu{
```

Comment:
I think this should be formatted as
```
     std::unique_ptr<wxMenu> census_menu
         {wxXmlResource::Get()->LoadMenu("census_menu_ref")
         };
```
probably.

I also realize now, thanks to #121, that this is not new code and I probably shouldn't have asked you to change it to use `unique_ptr`...